nodes/widgets/ui_form.html

```diff
@@ -60,6 +61,9 @@
             label: function () { return this.name || this.label || 'form' },
             labelStyle: function () { return this.name ? 'node_label_italic' : '' },
             oneditprepare: function () {
+                // Set passthru to `false` to prevent message pass through in Form widget
+                this.passthru = false
```

Comment:
Not sure this is required here if we're not presenting the user with an object, it would also require users to have opened the palette for a form before this works. 

Instead, this could be placed into `ui_form.js` and be hard-coded for all forms by doing `config.passthru = false`

Reply:
Ok, indeed if the user never opens the config panel, the variable will always be `true` based on previous configuration.

Will change to `ui_forms.js`